README.md

```diff
@@ -0,0 +1 @@
+Trials 1 Project
```

Comment:
Unfortunately, you have submitted an unprofessional README.m file. Please revise and redo a new one.

Reply:
Thank you for your valuable feedback on the descriptiveness of the README file. 👍 I would greatly appreciate it if you could provide specific examples or suggestions on how to make it more descriptive. For instance, if you could share a template link to follow or suggest essential elements that should be included in a descriptive README, it would be immensely helpful.

This information will not only help someone understand the areas that require improvement but also enable him/her to take actionable steps to enhance the readability and clarity of the README.

Please consider making your comment more descriptive. 🙏

Reply:
- [ ] Kindly do as the previous reviewer requested 🙏🏽  It is always advisable to give more contextual information when requesting someone to implement any tasks 🧑‍🏭 
In this case, you should for example inform the **student** of the importance of the documentation for any project 📓 That way, they would learn a new thing, not just blindly implementing your request ❤️

Reply:
> - [ ] Hello @JhonatanErosa, I wanted to bring to your attention that the changes requested by the previous reviewer have not been implemented yet. I kindly request you address those changes as soon as possible.👍When you explain the reasons behind a requested change, it helps the student understand the specific problem or improvement that needs to be addressed. By providing context and reasoning, you ensure that the student comprehends the purpose of the request.✔